src/library/trackmodel.h

```diff
@@ -1,54 +1,56 @@
-#ifndef TRACKMODEL_H
-#define TRACKMODEL_H
+#prargma once
 
-#include <QList>
-#include <QLinkedList>
 #include <QItemDelegate>
+#include <QLinkedList>
+#include <QList>
 #include <QtSql>
 
+#include "library/dao/settingsdao.h"
 #include "track/track.h"
 #include "track/trackref.h"
-#include "library/dao/settingsdao.h"
 
-/** Pure virtual (abstract) class that provides an interface for data models which
-    display track lists. */
+/// Pure virtual (abstract) class that provides an interface for data models
+/// which display track lists.
 class TrackModel {
   public:
     static const int kHeaderWidthRole = Qt::UserRole + 0;
     static const int kHeaderNameRole = Qt::UserRole + 1;
 
     TrackModel(QSqlDatabase db,
-               const char* settingsNamespace)
+            const char* settingsNamespace)
             : m_db(db),
               m_settingsNamespace(settingsNamespace),
               m_iDefaultSortColumn(-1),
               m_eDefaultSortOrder(Qt::AscendingOrder) {
     }
-    virtual ~TrackModel() {}
+
+    virtual ~TrackModel() {
+    }
 
     // These enums are the bits in a bitvector. Any individual column cannot
     // have a value other than 0, 1, 2, 4, or 8!
+    // clang-format off
     enum Capabilities {
```

Comment:
```suggestion
    enum class Capabilities : unsigned int {
```
If you use unsigned int as base class, you can drop the `u` suffix for the literals I think. 

Also, please use PascalCase for enum values. Enum class values are scoped, so you can drop the TRACKMODELCAPS prefix.

Reply:
Using unsigned int as base class doesn't help, unfortunately :/

Reply:
Adjusted the naming: https://github.com/mixxxdj/mixxx/pull/2810/commits/bc131a19d85c81af5fe632e7b8add4e249ae45e7#diff-bc279634b71150d9db535028f51ffd80